tests/integration/targets/cloud_discovery_providers_gcp_info/tasks/main.yml

```
@@ -0,0 +1,87 @@
+          - gcp_provider_info.objects[0].credential_preference.credential_type == "dynamic"
+          - gcp_provider_info.objects[0].tags.location == "site-1"
+
+    - name: Get GCP cloud discovery provider by ID
+      infoblox.universal_ddi.cloud_discovery_providers_info:
+          id: "{{ gcp_provider.id }}"
+      register: gcp_provider_info_by_id
+    - assert:
+        that:
+          - gcp_provider_info_by_id.objects | length == 1
+          - gcp_provider_info_by_id.objects[0].id == gcp_provider.id
+
+    - name: Get GCP cloud discovery provider by filter query
+      infoblox.universal_ddi.cloud_discovery_providers_info:
+        filter_query: "name=='{{ cloud_discovery_provider_gcp_name }}'"
+      register: gcp_provider_info_by_query
+    - assert:
+        that:
+          - gcp_provider_info_by_query.objects | length == 1
+          - gcp_provider_info_by_query.objects[0].name == cloud_discovery_provider_gcp_name
+
+    - name: Get GCP cloud discovery provider by tag filters
+      infoblox.universal_ddi.cloud_discovery_providers_info:
+        tag_filters:
+          location: "site-1"
+      register: gcp_provider_info_by_tag
+    - assert:
+        that:
+          - gcp_provider_info_by_tag.objects | length == 1
+          - gcp_provider_info_by_tag.objects[0].name == cloud_discovery_provider_gcp_name
+
+  always:
+     - name: "Delete a GCP cloud discovery provider"
+       infoblox.universal_ddi.cloud_discovery_providers:
+         name: "{{ cloud_discovery_provider_gcp_name }}"
+         provider_type: "Google Cloud Platform"
+         account_preference: "single"
+         credential_preference:
+           access_identifier_type: "project_id"
+           credential_type: "dynamic"
+         source_configs:
+           - credential_config:
+               access_identifier: "{{ config_access_id }}"
+         state: absent
+         ignore_errors: true
```

Incorrrect indentations for ignore errors

---

done